services/proto/src/teaclave_frontend_service.rs

```diff
@@ -1072,6 +1074,7 @@ impl std::convert::TryFrom<proto::FunctionOutput> for FunctionOutput {
     fn try_from(proto: proto::FunctionOutput) -> Result<Self> {
         let ret = Self {
             name: proto.name,
+            option: proto.option,
```

Comment:
In Python API, `option` is the 3rd argument. Do you think it's better to make them consistent?